scania-digital-design-system/tegel
# Feat/table: Add a property to control whether padding is applied to the expanded area of an expanded table row

## **Describe pull-request**  
Add a property to control whether padding is applied to the expanded area of an expanded table row.

## **Issue Linking:**  
- **GitHub:** https://github.com/scania-digital-design-system/tegel/issues/785

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Table -> "Expandable Rows" in Storybook.
2. Go to Docs and set "Expanded padding enabled" to false.
3. Expand a row and you will see that the padding is gone.

## **Checklist before submission**
- [x] Added e2e test

## **Suggested test steps**

## **Screenshot**
Desired result:
<img width="840" alt="no-padding" src="https://github.com/user-attachments/assets/a9f24308-65a7-4d9a-b77f-63cc08c6de86">
